.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
This file is needed?

Reply:
It's not required, but it helps to configure an editor of any contributor to right (for project) settings and prevent too long lines, whitespaces at line ends, no EOL symbols in files, etc. Like this: https://github.com/biola/Voight-Kampff/pull/41/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L54

It's pretty common standard, without OS or editor lock, so I don't see a problem with it, only help.